developer/messages/wrong-api-key.mdx

```diff
@@ -0,0 +1,17 @@
+---
+# this page is referenced in Leo, please do not rename without updating the reference in:
+# https://github.com/makeswift/cosmos/blob/master/packages/leo/src/lib/docs.ts
+
+title: You are not using the correct API key in your environment variables
+---
+
+## What this message means
+
+It seems like you are have a wrong API key in your `MAKESWIFT_SITE_API_KEY` environment variable. 
```

Comment:
The API key doesn't have to necessarily come from environment variables or be named `MAKESWIFT_SITE_API_KEY`. This is what we document in our docs, but all users need to do is pass a value to the client constructor and API handler. If we're going to reference specific environment variable names I think we should do it while calling out that it's one way, or our documented way, while the problem would be cause by "the wrong API key", not "the wrong environment variable".

Reply:
I think the title and [the panel message](https://www.figma.com/design/DUBoFzjgADywP2h8ZXLmyo/App-v2?node-id=17093-324375&t=C6JjYxT6oE40zjTu-4) should change as well. Maybe just:

> You are not using the correct API key

cc @andrewreifman

Reply:
Yeah, I like that better.

Reply:
We could also say something like "API key mismatch", "The API key configured in the host doesn't match the API key of this Makeswift site", etc.

Reply:
I'm good with @migueloller's suggestion. I tested it in Figma and it's not too much text either.